homeassistant/components/persistent_notification/notify.py

```diff
@@ -0,0 +1,32 @@
+"""Persistent notification platform for notify integration."""
+from typing import Any, Optional
+
+from homeassistant.components.notify import BaseNotificationService
+from homeassistant.core import HomeAssistant
+
+from . import ATTR_MESSAGE, ATTR_TITLE, DOMAIN, SERVICE_CREATE
+
+
+async def async_get_service(
+    hass: HomeAssistant, config: dict, discovery_info: Optional[dict] = None
+) -> BaseNotificationService:
+    """Get the persistent notification service."""
+    return PersistentNotifyPlatform(hass)
+
+
+class PersistentNotifyPlatform(BaseNotificationService):
+    """Implement the notification service for the persistent notification notify platform."""
+
+    hass: HomeAssistant
+
+    def __init__(self, hass: HomeAssistant):
+        """Initialize the service."""
+        self.hass = hass
+
+    async def async_send_message(self, message: str, **kwargs: Any) -> None:
+        """Create persistent notification from message."""
+        payload = {ATTR_MESSAGE: message}
+        if ATTR_TITLE in kwargs:
+            payload[ATTR_TITLE] = kwargs[ATTR_TITLE]
+
+        await self.hass.services.async_call(DOMAIN, SERVICE_CREATE, payload)
```

Comment:
I call the service here via the service registry. I'm not sure if importing the underlying function from __init__.py and using that directly would be better. Thoughts?